### PR TITLE
Moreinfo

### PR DESCRIPTION
**Pull request by federicomarini**
Added details for installing the package in the welcome README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,27 @@ Current contributors include:
 
 [![Figure 1. _iSEE_ uses a customisable multi-panel layout.][Figure1]](https://f1000research.com/articles/7-741/v1)
 
+## Installation
+
+_iSEE_ can be easily installed from Bioconductor using `BiocManager::install()`:
+
+``` r
```
Comment (LTLA) [inline README.md]: Shouldn't be a space here.

File: README.md [hunk 2]
```diff
@@ -30,6 +30,27 @@ Current contributors include:
...
+if (!requireNamespace("BiocManager", quietly = TRUE))
+    install.packages("BiocManager")
+BiocManager::install("iSEE")
+```
+
+Optionally, if you want to install the development version from GitHub, you can use:
```
Comment (LTLA) [inline README.md]: Why are we encouraging this?

Comment (federicomarini) [reply README.md]: I was split for that
- on one side, it is the GitHub version, so maybe someone would want it (and would like to see the instructions)
- OTOH, advanced users could know the drill anyway
Should this just be gone?

File: README.md (hunk 1 above)
Comment (federicomarini) [reply README.md]: I saw it in other repos, works in both cases ;)

File: README.md (hunk 2 above)
Comment (LTLA) [reply README.md]: Yes.

Comment (federicomarini) [reply README.md]: Ok, going away then.

**Pull request merged**